manifests/n/NotroDev/SkEditor+/1.7.8/NotroDev.SkEditor+.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: NotroDev.SkEditor+
+PackageVersion: 1.7.8
+InstallerType: msi
+ProductCode: '{29E3F9DC-A3F3-499B-A230-3C9F02903C82}'

```

Comment:
```suggestion
ProductCode: '{29E3F9DC-A3F3-499B-A230-3C9F02903C82}'
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
```

Reply:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/08860f5e-3f96-4319-8103-ccdd789626a1)

Reply:
I forgot to try on a fresh vm without it, sorry